flood_adapt/object_model/hazard/event/synthetic.py

```diff
@@ -114,3 +141,45 @@ def add_wind_ts(self):
                 "timeseries"
                 "."
             )
+
+    @staticmethod
```

Comment:
Ik snap nog niet wanneer @staticmethod wel of niet moet. Dus ik vertrouw erop dat jij dat wel scherp hebt :)